plenoptic/metric/perceptual_distance.py

```diff
@@ -186,6 +181,10 @@ def ssim(img1, img2, weighted=False, dynamic_range=1):
     else:
         mssim = (map_ssim*weight).sum((-1, -2)) / weight.sum((-1, -2))
 
+    if min(img1.shape[2], img1.shape[3]) < 11:
```

Comment:
shouldn't this warning just be raised in `_ssim_parts`? It looks like it gets raised by all of `ssim`, `ssim_map`, and `ms_ssim`, so may as well reduce the repeating (also, raising it closer to where the relevant bit happens in the code makes it easier to read)

Reply:
The warnings for SSIM and MS-SSIM are different. I initially tried to raise a warning in `_ssim_parts`, and catch the warning and raise another warning in `ms_ssim`, but then I think it's cleaner to write them separately.

Reply:
ah I missed the difference between them. That makes sense to me then.